src/net/http/fs.go

```diff
@@ -264,7 +264,7 @@ var errNoOverlap = errors.New("invalid range: failed to overlap")
 
 // if name is empty, filename is unknown. (used for mime type, before sniffing)
 // if modtime.IsZero(), modtime is unknown.
-// content must be seeked to the beginning of the file.
+// content must be sought to the beginning of the file.
```

Comment:
Some people might find (incorrect form) `"seeked"` more easy to read in code, understood if you prefer me to revert this.